var/spack/repos/builtin/packages/coreutils/package.py

```diff
@@ -30,6 +32,7 @@ class Coreutils(AutotoolsPackage, GNUMirrorPackage):
           sha256='5878894375a8fda98150783430b30c0b7104899dc5522034ebcaf8c961183b7e')
 
     build_directory = 'spack-build'
+    executables = [r'^ls$']
```

Comment:
I'm a bit hesitant to detect coreutils like this, since:
- it's not a common dependency (thus the use of detection is limited)
- 2 out of 3 dependents use the `md5sum` executable

Reply:
OK fine, changed to md5sum. Thanks for reviewing.